<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerBrowser</class>
 <widget class="QMainWindow" name="ServerBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerBrowser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabConnect">
     <attribute name="title">
      <string>Connect</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nickname:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNick">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>18</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Player</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leIP">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>48</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePort">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>78</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5192</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePassword">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>108</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbWindowed">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Windowed</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConnect">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>46</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFavourites">
     <attribute name="title">
      <string>Favourites</string>
     </attribute>
     <widget class="QPushButton" name="btnAddFav">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>520</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Server</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabServers">
     <attribute name="title">
      <string>Servers</string>
     </attribute>
     <widget class="QPushButton" name="btnConnect2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="twServers">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>771</width>
        <height>511</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="wServers" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefresh">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>520</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFeatured">
     <attribute name="title">
      <string>Featured</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabSponsors">
     <attribute name="title">
      <string>Sponsors</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
